files/galaxy/tpv/tools.yml

```diff
@@ -1338,6 +1338,7 @@ tools:
       _GALAXY_JOB_TMP_DIR: '/tmp'
     params:
       singularity_run_extra_arguments: "--env GENEMARK_PATH=/usr/local/tools/genemark/etp.for_braker/bin/gmes/"
+      singularity_no_mount: null
```

Comment:
null and not false?

Reply:
This variable seems to take a list of strings, and the org defines it as `null` to overwrite the defaults (`\tmp`). Unfortunately, there is no reference to this variable anywhere in the code base, and it lacks documentation. So, I think it will just add `--no-mount null` in the final populated singularity command. Let's try it out and see what happens.